src/components/CarTypeBar/CarTypeBar.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./CarTypeBar.scss";
+const CarTypeBar = ({ carType, dispatchCarsByType }: any) => {
+  return (
+    <div className="carTypeBar">
+      <div className={carType === "sedan" ? "activeCarType" : ""}>
+        <h4
```

Comment:
Not actionable: Since this progressbar sort of thing that we are building is a purely synthetic element we should be using span tags here, and not h4. It ideally should have been a list.